e2e-tests/specs/all.e2e.js

```diff
@@ -274,23 +272,35 @@ describe('Hide CTA(s):', async () => {
         const exists = await $('#ac578dee-f01b-3ea0-a282-2116619e4251 .consonant-BtnInfobit--cta').isDisplayed();
         expect(exists).toEqual(false); 
     });
+    it('MWPW-130075: Card with hidden CTA should be clickable', async () => {
+        const cloneConfig = structuredClone(config);
+        cloneConfig.hideCtaIds = ['ac578dee-f01b-3ea0-a282-2116619e4251'];
+        const state = btoa(JSON.stringify(cloneConfig));
+        const url = `${serverPath}/html/e2e/MWPW-126169.html?state=${state}`;
+        await browser.url(url);
+        await browser.setTimeout({ script: 50000 });
+        const hiddenCTA = await $('#ac578dee-f01b-3ea0-a282-2116619e4251 .consonant-LinkBlocker').isDisplayed();
+        expect(hiddenCTA).toEqual(true); 
+    });
     it('MWPW-128711: ALL CTAs should be hidden', async () => {
-        const cloneConfig = { ...config };
+        const cloneConfig = structuredClone(config);
         cloneConfig.collection.collectionButtonStyle = 'hidden';
         const state = btoa(JSON.stringify(cloneConfig));
         const url = `${serverPath}/html/e2e/MWPW-126169.html?state=${state}`;
         await browser.url(url);
         await browser.setTimeout({ script: 50000 });
+        await browser.pause(5000);
```

Comment:
@gmirijan is this browser pause of 5 seconds necessary for the e2e test or just a left over while developing? Best would be if we could remove it to remove it since it'll add 5 seconds extra to our build process.

Reply:
It is a left over and should be removed.